src/DotNetWorker.Core/Http/DefaultHttpRequestDataFeature.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Functions.Worker.Http
+{
+    internal class DefaultHttpRequestDataFeature : IHttpRequestDataFeature
+    {
+        private const string httpTriggerString = "httpTrigger";
```

Comment:
nit: Can you check the naming convention we use for constants in other files this repo. I believe we use Pascal casing (`HttpTriggerString`)

Reply:
Pretty sure it's pascal